### DISCUSSION TITLE
Is it a problem to use `AfterCreateTable` hook for FKs?

### DISCUSSION THREAD

**Discussion by frederikhors** [uptrace/bun#379] [Q&A] [answered]
I know it's a stupid question but maybe no question is.

In the docs [there is an example](https://bun.uptrace.dev/guide/query-create-table.html#hooks) of creating foreign key using hook `BeforeCreateTable`, is it a problem if I'm using `AfterCreateTable` instead?

**Comment by vmihailenco** [comment, marked as answer]
Yes, `AfterCreateTable` is run after the query is executed so can't modify the query in it.
